.changes/next-release/bugfix-828c2cd6-7fc0-4136-a0f3-68f41725eb85.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "bugfix",
+  "description" : "Fix Rider locking up when right clicking a Lambda in the AWS Explorer with a dotnet runtime in 2021.1"
+}
```

Comment:
silly nit: new line / no new line on these files?

Reply:
Oh, after the end? We need to fix that in the changelog generator script